lib/arraySorting.js

```diff
@@ -1,11 +1,14 @@
-const names = ["Alice", "Bob", "Jeremy", "Sam", "Henry", "Sarah", "Ashely"];
+const name = ["Alice"];
 // const names = [];
 
 function Printname() {
-  names.forEach((name) => {
-    console.log("Hello my name is" + " " + name);
-    return "Hello may name is" + " " + name;
-  });
+  //   names.forEach((name) => {
```

Comment:
Not sure why we'd want a loop really the following should suffice for how you're test is defined ~

` return "Hello my name is" + " " + name"`
or better yet
```
return `Hello my name is ${name}` 
```

This would cause the test to pass, if you pass the proper name within your test.
Question why did we use a loop anyways?

Reply:
For the test I'm doing I have to return an array of names. So I'm basically taking an array and then adding 'Hello my name is' for each name in the array.

Reply:
Gotcha so then you want to use `map` here. Something like so ~
```
return names.map((name) => {
    return `Hello my name is ${name}`
}
```

Reply:
Your names array should be defined as a paramater to the Printname function.